wiki/Help_Centre/en.md

```diff
@@ -55,29 +51,55 @@ Good information to include this email would be:
 
 If this succeeds, you will be able to reset your own password without the assistance of the support staff.
 
-If this does not work for you, please send us a support ticket immediately to [accounts@ppy.sh](mailto:accounts@ppy.sh) with as much identifying information as possible that will help us establish you as the owner of your account.
+If this does not work for you, please send us a support ticket immediately to [accounts@ppy.sh](mailto:accounts@ppy.sh) with as much identifying information as possible that will help us identify you as the owner of your account.
 
 Emailing from your registration address is a huge bonus here, as well as any other information you can provide that only the holder of the account would know.
 
 Please do not include replay or score data in your ticket - we cannot use this to verify your ownership of the account.
 
+#### What is this 'Bancho authentication error' I keep on receiving?
+
+**This typically happens when there are networking issues between you and our servers.**
+
+There's a few things we can try to resolve this issue:
+
+##### Have you entered your password correctly?
```

Comment:
please unify the headings throughout the article, they should be written either from the reader's point of view ("I can't ..."), or the writer's ("Have you ... ?"), but not from both at the same time (example: here and a couple of lines above)

Reply:
I'm not sure if some of the changes I made are the best way to go about it, but now all level 5 headings are from the writer's POV (and the level 4 headings are reader's POV) 

If you meant all of them(4 & 5 combined) should be either writers or readers pov, I personally don't think that would work very well. They work pretty well as they are now in my opinion.